app/src/main/kotlin/com/github/gotify/CoilInstance.kt

```diff
@@ -69,7 +74,9 @@ object CoilInstance {
         context: Context,
         sslSettings: SSLSettings
     ): Pair<SSLSettings, ImageLoader> {
-        val builder = OkHttpClient.Builder()
+        val builder = OkHttpClient
+            .Builder()
+            .authenticator(BasicAuthAuthenticator())
```

Comment:
> I've used an authenticator instead of an interceptor.
The difference is that an interceptor is used on every single call which is not needed in our case.
The authenticator only is called when a request throws a 401 so we try again with basic auth.

I'd say this is a drawback, now there need to be two requests, one fails and the next one get the authentication. I'd say having an interceptor that does nothing isn't really a problem. I'd rather only execute one requests.

Reply:
Alright, I will change that

Reply:
Done in 7d6399b087a54702842a7953a055de8c939590e8